src/test/components/Home.test.js

```diff
@@ -99,41 +92,18 @@ describe('app/Home', function() {
     // There is an outstanding question to the WebChannel
     expect(listeners.length).toBeGreaterThan(0);
 
-    // Respond from the browser that the menu button is available.
-    triggerResponse({
-      type: 'STATUS_RESPONSE',
-      menuButtonIsEnabled: true,
-      requestId: getLastRequestId(),
-    });
-
-    // The UI should update for the record instructions, which is an async
-    // handle of the WebChannel message.
-    const instructions = await findByTestId('home-record-instructions');
-
-    expect(instructions).toMatchSnapshot();
-  });
-
-  // This test's assertions are that it can find elements through getByTestId.
-  // eslint-disable-next-line jest/expect-expect
-  it('will switch to recording instructions when enabling the popup', async () => {
-    const { triggerResponse, getLastRequestId } = mockWebChannel();
-    const { findByTestId, getByText } = setup(FIREFOX);
-
-    // Respond back from the browser that the menu button is not yet enabled.
+    // Respond from the browser that the WebChannel does not exist.
     triggerResponse({
-      type: 'STATUS_RESPONSE',
-      menuButtonIsEnabled: false,
-      requestId: getLastRequestId(),
+      errno: 2, // ERRNO_NO_SUCH_CHANNEL
+      error: 'No such channel',
     });
-    await findByTestId('home-enable-popup-instructions');
 
-    fireFullClick(getByText('Enable \u2068Firefox Profiler\u2069 Menu Button'));
+    // The UI should update to include a note about the WebChannel being unavailable,
+    // which is an async handler of the WebChannel message.
+    const webChannelUnavailableMessage = await findByTestId(
+      'home-web-channel-unavailable-message'
+    );
 
```

Comment:
You can avoid using a "test id" by using `findByText`, for example:
```
await findByText(/This profiler instance was unable to connect to the WebChannel/);
```
And then you can remove the test id attribute in the app code too.